### DISCUSSION TITLE
Integration and/or compatibility with Watchtower?

### DISCUSSION THREAD

**Discussion by dataprolet** [nextcloud/all-in-one#3797] [Questions] [answered]
I couldn't find any obvious documentation on how Nextcloud-AIO utilizes or integrates with an existing Watchtower container? I see that most containers have the _com.centurylinklabs.watchtower.enable_ set to _false_ but not all. Is it fine to have Watchtower running? Is it intended? Is the master container starting a Watchtower container anyway unless there is an existing one? Instructions unclear.

**Comment by szaimen** [comment, marked as answer]
AIO has its own automatic update mechanism which you can enable from the AIO interface by enabling daily backups. That is why watchtower is disabled for them. Only the mastercontainer itself is updated via an aio-watchtower container which is done automatically as well if you enable automatic updates while enabling daily backups.

**Reply by dataprolet** [reply]
But without enabling daily backups there is no aio-watchtower container. But I already have a watchtower container that will (try to) update the mastercontainer. Is that supported?

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one#how-to-enable-automatic-updates-without-creating-a-backup-beforehand id you dont want to emable daily backups

**Reply by dataprolet** [reply]
I'm sorry, but I don't want to know how to update. All I want to know is whether it's a problem to have separate watchtower container running because the nextcloud-aio-mastercontainer does not have _com.centurylinklabs.watchtower.enable_ set to _false_.

**Reply by Zoey2936** [reply]
you can run a separate watchtower container

**Reply by dataprolet** [reply]
Thanks you!
